.github/workflows/release.yml

```
@@ -131,3 +132,62 @@ jobs:
+        with:
+          repository: beaver-lodge/beaver
+          path: beaver
+      - uses: actions/checkout@v3
+        name: Check-out kinda
+        with:
+          repository: beaver-lodge/kinda
+          path: kinda
+      - name: Build ARM
+        continue-on-error: true
+        if: ${{ matrix.image.platform == 'linux/arm64/v8' }}
+        run: docker run --platform ${{ matrix.image.platform }} -v $PWD:/src -w /src/beaver jackalcooper/beaver-livebook-${{ matrix.image.suffix }}:latest bash scripts/build-for-publish.sh
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:55: Double quote to prevent globbing and word splitting

(shellcheck)